apps/accounts/verify.py

```diff
@@ -0,0 +1,35 @@
+# Don't import any django stuff here.
+
+import requests
+
+
+def fxa_identify(code, config=None):
+    """Get an FxA profile for an access token. If idenficiation fails return
+    an empty dict."""
+    token = get_fxa_token(code, config).get('access_token')
+    if token:
+        return get_fxa_profile(token, config)
+    else:
+        return {}
```

Comment:
the main thing I'd like to see is 500 handling (+ logging) which I think would be a little easier with exceptions but still doable with an empty dict